dist/css/react-calendar.css

```diff
@@ -215,7 +215,6 @@
     right: -50px;
     top: -50px;
     -webkit-transform: rotate(30deg);
-        -ms-transform: rotate(30deg);
```

Comment:
I would assume this was part of the build, I made no manual changes to the dist folder, just re-ran the build to double check.